README.md

```diff
@@ -1,87 +1,107 @@
 
-* `~/your_workspace/devstack-vagrant/`
-
-## Setup DevStack
-
-Provision DevStack.
+### Install OpenStack CLI tools
 
+- heat >= 0.9.0
+- nova >= 3.2.0
 ```
-cd ~/your_workspace/devstack-vagrant/
-DEVSTACK_MEM=10024 DEVSTACK_CPUS=6 vagrant up --provider libvirt
+ sudo pip install -U --force python-heatclient
+
+ sudo pip install -U --force python-novaclient
 ```
 
-Download custom image from Google drive: <https://drive.google.com/file/d/0B6KOuPCy8tK1aDRCaE16N1hkcUU/view>
-Uncompress it and upload to vgrant host
 
-```
-unxz centos7-docker.qcow2.xz
-scp -i devstack/.vagrant/machines/devstack/libvirt/private_key centos7-docker.qcow2 vagrant@192.168.123.100:/tmp/centos7-docker.qcow2
-```
-Upload custom Centos image to OpenStack.
+### Configure Openstack CLI tools
 
-```
-cd ~/your_workspace/devstack-vagrant/
-vagrant ssh
+ Please get your OpenStack credential and modify the variables in the following files:
 
-sudo su - stack
-. devstack/openrc admin admin
+ - **config-default.sh** Sets all parameters needed for heat template.
+ - **openrc-default.sh** Sets environment variables for communicating to OpenStack. These are consumed by the cli tools (heat, nova).
 
-glance image-create --name centos7 --disk-format qcow2 --container-format bare --file /tmp/centos7-docker.qcow2 --is-public True
-```
+### Get kubectl
+
+If you already have the kubectl, you can skip this step.
 
-Create and register a keypair.
+kubectl is a command-line program for interacting with the Kubernetes API. The following steps should be done from a local workstation to get kubectl.
+Download kubectl from the Kubernetes release artifact site with the curl tool.
 
+The linux kubectl binary can be fetched with a command like:
 ```
-ssh-keygen -f ~/.ssh/id_rsa -P ''
-nova keypair-add keypair1 --pub-key ~/.ssh/id_rsa.pub
+$ curl -O https://storage.googleapis.com/kubernetes-release/release/v1.2.0/bin/linux/amd64/kubectl
 ```
 
-## Deploy Kubernetes cluster
-
+Make kubectl visible in your system.
 ```
-admin_tenant_id=$(openstack project show admin -f value -c id)
-
-image_id=$(openstack image show centos7 -f value -c id)
-dns_server=10.0.238.34 # Change it to match your environment
-
-git clone http://estscm1.intern.est.fujitsu.com/wlm/heat-docker-kubernetes.git
-cd heat-docker-kubernetes
-
-heat stack-create \
-     -P external_network=public \
-     -P ssh_key_name=keypair1 \
-     -P server_image=${image_id} \
-     -P master_flavor=m1.small \
-     -P minion_flavor=m1.small \
-     -P number_of_minions=1 \
-     -P max_number_of_minions=1 \
-     -P dns_nameserver=$dns_server \
-     --timeout 60 \
-     --template-file kubecluster.yaml \
-     kubernetes_stack
+sudo cp kubectl /usr/local/bin
 ```
 
-## Verification
+### Prepare Openstack image
 
-From your client computer:
+You need Centos image for Openstack with installed Docker and docker-bootstrap service. More info you can find in separate
```

Comment:
The provisioning works on any operating system that has a Docker and Docker-bootstrap service installed. 

You can download such a prepared image from ...

If you want to build your own image you can use this project .... bla bla

More info....